client/spec/multicontent_spec.js

```diff
@@ -6,13 +6,14 @@ var workspace = require('./helpers/workspace'),
     content = require('./helpers/content'),
     multicontent = require('./helpers/multicontent');
 
-describe('multi content view', function() {
+// todo(petr): those features should go to list
+xdescribe('multi content view', function() {
```

Comment:
OK, if it needs more refactoring, then it's fine to leave the TODO here (just make sure that there is a ticket for that).